docs/modules/ROOT/pages/performance/sql_and_scan_query_pref_comp.adoc

```diff
@@ -0,0 +1,48 @@
+= SQL Fields Query and Scan Query Performance Comparision
+Prasad Kommoju <pkommoju@futurewei.com>
+2021-11-30
+:toc: right
+:imagesdir: ../../images
+
+
+== Introduction
+In Alcor there are some cases where queries are made on the cases using a field other than the key. Since Ignite is the KV database, these types of queries will require full scan (linear search) of the cache in question and this introduces big latencies as the number of entries in the cache grows.
+
+Ignite supports secondary keys, called QuerySqlFields which will use a tradional B+Tree indexes which can answer point lookup and also range, minumum, maximum, greater than, and less than queries mush faster.
+
+This report is about comparing the performance of the queries on non-key fields using Ignite's SCAN query and through SQL queries.
+
+== Test setup
+Operations are on NodeInfo object. Benchmark tool, DPM, NCM and NMM run on 10.213.43.161.
+Ignite is ru on 10.213.43.163 and 10.213.43.164.
+Number of entries in both SQL and Scan are 1M, 1000 entries are queried on.
```

Comment:
@pkommoju I think it's not good to show ip addresses here.

Reply:
removed IP addresses.